src/index.html

```diff
@@ -1,17 +1,156 @@
+          >
+        </div>
+        <div class="formField">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+            pattern="\w+"
+          >
+        </div>
+        <div class="formField">
+          How old are You?
+          <input
+            type="number"
+            name="age"
+            min="1"
+            max="100"
+            value="12"
+          >
+        </div>
+        <div class="formField">
+          Full date of birth:
+          <input type="date" name="birthDate" >
+        </div>
+        <label class="form">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="toa"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend class="blockTitle">Registration</legend>
+        <div class="formField">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </div>
+        <div class="formField">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="6"
+          maxlength="32"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend class="blockTitle">An interesting fact about you!</legend>
+        <div class="formField">
+          Do you love cats?
+          <input
+          type="radio"
+          name="cats"
+          id="Yes"
+          >
+          <label for="cats">Yes</label>
+          <input
+          type="radio"
+          name="cats"
+          id="No"
+          >
+          <label for="cats">No</label>
+        </div>
+        <div class="formField">
+          What is your favorite color?
+          <input type="color" name="color" >
+        </div>
+        <div class="formField">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="bedTime"
+          step="2"
+          >
+        </div>
+        <div class="formField">
+          <label for="cars">What are your favorite brand of cars:</label>
+          <select
+          name="cars"
+          id="cars"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </div>
+        <div class="formField">
+          How do you rate our work?
+          <input
+          type="range"
+          name="ratind"
+          min="1"
+          max="10"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="formBlock">
+        <legend>Additional info:</legend>
+        <div class="formField">
+          Comments:
+          <textarea
+          name="comments"
+          cols="20"
+          rows="2"
+          >
+          </textarea>
```

Comment:
[Checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) the 8th point. 
There shouldn't be any spaces in text area:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/52370890/220177711-40d61a03-a29c-4b8e-819f-e5f34845f893.png">

Hint: you should put open and close tags on the same level, cause all indents between counts like spaces